app/build.gradle

```diff
@@ -4,7 +4,7 @@ android {
     buildToolsVersion '23.0.2'
     defaultConfig {
         applicationId 'fr.neamar.kiss'
-        minSdkVersion 10
+        minSdkVersion 11
```

Comment:
I wrote this line in doc's proveider.
  if (android.os.Build.VERSION.SDK_INT > Build.VERSION_CODES.GINGERBREAD_MR1) 
It will not run on lower from 11 and definitely will not crash.